drafts/use-cases/1.1.security-considerations.md

```diff
@@ -10,23 +10,30 @@ There are several possibilities here. One of them would be to use a *@graph* not
 
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@graph": [
         {
             "@id": "/api",
-            "operation": [
-                {
-                    "@type": "hydra:Operation",
-                    "title": "List events",
-                    "method": "GET",
-                    "returns": "hydra:Collection"
-                },
-                {
-                    "@type": "hydra:Operation",
-                    "title": "Create new event",
-                    "method": "POST",
-                    "expects": "schema:Event"
-                }
-            ]
+            "@type": "hydra:EntryPoint",
+            "#events": {
```

Comment:
Please let's get rid of `#events`. It won't work unless `@vocab` is specified in the context. If the vocab is set to schema.org or Hydra, it would make this property part of those vocabularies. Certainly not what you intended. Could we change it to `myvocab:events` or something similar for the time being?